pkg/agent/secondarynetwork/podwatch/controller_test.go

```diff
@@ -320,11 +496,37 @@ func TestPodControllerAddPod(t *testing.T) {
 		_, err = podController.netAttachDefClient.NetworkAttachmentDefinitions(testNamespace).Create(context.Background(), network2, metav1.CreateOptions{})
 		require.NoError(t, err, "error when creating test NetworkAttachmentDefinition")
 		assert.NoError(t, podController.handleAddUpdatePod(pod))
+
+		infos := podController.podCache.GetAllCNIConfigInfoPerPod(podName, testNamespace)
+		assert.Equal(t, 1, len(infos))
+		config1, _ := netdefutils.GetCNIConfig(network1, "")
+		config2, _ := netdefutils.GetCNIConfig(network2, "")
+		savedCNIConfig.Interfaces = map[string]*cnipodcache.InterfaceInfo{
+			"eth10": &cnipodcache.InterfaceInfo{
+				NetworkType: sriovNetworkType,
+				CNIConfig:   config1,
+			},
+			"eth11": &cnipodcache.InterfaceInfo{
+				OVSPortUUID: ovsPortUUID,
+				NetworkType: vlanNetworkType,
+				CNIConfig:   config2,
+			},
+		}
+		assert.True(t, reflect.DeepEqual(&savedCNIConfig, infos[0]))
```

Comment:
I thought `assert.Equal` already used `reflect.DeepEqual` to check for comparison?

https://github.com/stretchr/testify/blob/f97607b89807936ac4ff96748d766cf4b9711f78/assert/assertions.go#L58

Reply:
You are right. Changed to use `assert.Equal`.